<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>333</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ViajeFácil - Nego Space</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Usuario:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string>&amp;Login</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Owners</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Negos</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_2"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_3">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Oficinas</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_3"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_4">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Peticiones</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QListWidget" name="listWidget_4"/>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_5">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>572</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuCrear">
    <property name="title">
     <string>&amp;Crear</string>
    </property>
    <addaction name="actionCreOwner"/>
    <addaction name="actionCreNego"/>
    <addaction name="actionCreOficina"/>
    <addaction name="actionCrePeticion"/>
    <addaction name="separator"/>
    <addaction name="actionCreUsuario"/>
   </widget>
   <widget class="QMenu" name="menuBorrar">
    <property name="title">
     <string>&amp;Borrar</string>
    </property>
    <addaction name="actionBorOwner"/>
    <addaction name="actionBorNego"/>
    <addaction name="actionBorOficina"/>
    <addaction name="actionBorPeticion"/>
   </widget>
   <widget class="QMenu" name="menuModificar">
    <property name="title">
     <string>&amp;Modificar</string>
    </property>
    <addaction name="actionModOwner"/>
    <addaction name="actionModNego"/>
    <addaction name="actionModOficina"/>
    <addaction name="actionModPeticion"/>
   </widget>
   <widget class="QMenu" name="menuConsultas">
    <property name="title">
     <string>C&amp;onsultas</string>
    </property>
    <addaction name="actionLog_de_Peticiones"/>
    <addaction name="actionInforme_Owners_que_mejor_consumen_Negos"/>
   </widget>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Ayuda</string>
    </property>
    <addaction name="actionSobre_ViajeFacil"/>
   </widget>
   <addaction name="menuCrear"/>
   <addaction name="menuModificar"/>
   <addaction name="menuBorrar"/>
   <addaction name="menuConsultas"/>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreNego">
   <property name="text">
    <string>&amp;Nego</string>
   </property>
  </action>
  <action name="actionCreOficina">
   <property name="text">
    <string>O&amp;ficina</string>
   </property>
  </action>
  <action name="actionCrePeticion">
   <property name="text">
    <string>&amp;Petición</string>
   </property>
  </action>
  <action name="actionCreOwner">
   <property name="text">
    <string>&amp;Owner</string>
   </property>
  </action>
  <action name="actionBorOwner">
   <property name="text">
    <string>&amp;Owner</string>
   </property>
  </action>
  <action name="actionBorNego">
   <property name="text">
    <string>&amp;Nego</string>
   </property>
  </action>
  <action name="actionBorOficina">
   <property name="text">
    <string>O&amp;ficina</string>
   </property>
  </action>
  <action name="actionBorPeticion">
   <property name="text">
    <string>&amp;Petición</string>
   </property>
  </action>
  <action name="actionModOwner">
   <property name="text">
    <string>&amp;Owner</string>
   </property>
  </action>
  <action name="actionModNego">
   <property name="text">
    <string>&amp;Nego</string>
   </property>
  </action>
  <action name="actionModOficina">
   <property name="text">
    <string>O&amp;ficina</string>
   </property>
  </action>
  <action name="actionModPeticion">
   <property name="text">
    <string>&amp;Petición</string>
   </property>
  </action>
  <action name="actionCreUsuario">
   <property name="text">
    <string>&amp;Usuario</string>
   </property>
  </action>
  <action name="actionLog_de_Peticiones">
   <property name="text">
    <string>&amp;Log de Peticiones</string>
   </property>
  </action>
  <action name="actionInforme_Owners_que_mejor_consumen_Negos">
   <property name="text">
    <string>&amp;Informe - Owners que mejor consumen Negos</string>
   </property>
  </action>
  <action name="actionSobre_ViajeFacil">
   <property name="text">
    <string>Sobre ViajeFácil</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>pushButton_2</tabstop>
  <tabstop>listWidget</tabstop>
  <tabstop>listWidget_2</tabstop>
  <tabstop>listWidget_3</tabstop>
  <tabstop>listWidget_4</tabstop>
  <tabstop>lineEdit_2</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_5</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
